dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java

```diff
@@ -112,12 +114,19 @@ public Entry parseWorkflowContent(Entry entry, String filepath, String content,
                 entry.setDescription(description[0]);
             }
         } catch (WdlParser.SyntaxError syntaxError) {
-            LOG.info("Invalid WDL file.");
+            LOG.error(MessageFormat.format("Unable to parse WDL file {0}", filepath), syntaxError);
+            clearMetadata(entry);
         }
 
         return entry;
     }
 
+    private void clearMetadata(Entry entry) {
+        entry.setAuthor(null);
+        entry.setEmail(null);
+        entry.setDescription(null);
```

Comment:
In lieu of the validation in develop, maybe we can just say something akin to "syntax error, please check your WDL file" here